src/fidesops/service/connectors/saas_connector.py

```diff
@@ -49,14 +51,15 @@ def get_authenticated_request(
         self, request_params: SaaSRequestParams
     ) -> PreparedRequest:
         """Returns an authenticated request based on the client config and incoming path, query, and body params"""
-        (path, params, data) = request_params
-        req = Request(url=f"{self.uri}{path}", params=params, data=data).prepare()
+        (method, path, params, data) = request_params
+        req = Request(
+            method=method, url=f"{self.uri}{path}", params=params, data=data
+        ).prepare()
         return self.add_authentication(req, self.client_config.authentication)
 
-    def get(self, request_params: SaaSRequestParams) -> Response:
+    def send(self, request_params: SaaSRequestParams) -> Response:
         """Builds and executes an authenticated GET request"""
```

Comment:
Can you update docstring to be more accurate?

Reply:
```Builds and executes an authenticated request. The HTTP method is determined by the request_params.```